src/procedures/ReserveSecurityToken.ts

```diff
@@ -18,12 +18,23 @@ import { findEvents } from '../utils';
 
 const { bigNumberToDate } = conversionUtils;
 
+/**
+ * - Procedure that reserve the token ticker to an specific owner
+ *
+ * ***Note once the token ticker is registered to its owner then no other issuer can claim its ownership***
```

Comment:
I think this is implied in the fact that it's a reservation, so it shouldn't need to be specified